lib/marionette/element.js

```diff
@@ -306,9 +306,11 @@
      */
     size: function size(callback) {
       var cmd = {
-        name: 'getElementSize'
+        name: 'getElementRect'
       };
-      return this._sendCommand(cmd, 'value', callback);
+      var resp = this._sendCommand(cmd, 'value', callback);
+      console.log(resp);
+      return resp;
```

Comment:
Shouldn't the conversion from the `getElementRect` response to the backwards-compatible `size` response be here? Same thing re: `location` below